### ISSUE TITLE
Exclamation mark used as new/old email address internal separator is displayed to user

### ISSUE THREAD

**Issue by mkordas** [yegor256/netbout#852]
##### Scenario 1:
User without email address at `acc/index` presses save without specifying any value. Empty email is correctly considered as invalid, but message displayed to user is `Email '!' is invalid`

##### Scenario 2:
User with assigned email address (e.g. current@email.com) at `acc/index` presses save without specifying any value. New email that is empty is correctly considered as invalid, but message displayed to user is `Email 'current@email.com!' is invalid`

**Comment by dmzaytsev** [comment]
@karato it's a valid bug

**Comment by karato** [comment]
> @karato it's a valid bug

@dmzaytsev tagged this issue with "bug"

**Comment by karato** [comment]
@mkordas I added milestone `3.1` to this issue, let me know if there has to be something else

**Comment by karato** [comment]
@mkordas thank you for the ticket reported, I topped your account for 15 mins, transaction `566ac1923c54100b1c000022`

**Comment by karato** [comment]
@original-brownbear do this task

**Comment by original-brownbear** [comment]
@dmzaytsev I think 442b8841ddf1aa0b25605960885461d1092be489 effectively turns this into a duplicate of #/629, right?
I mean in a sense fixing #629 fixes this, the issue as described by you cannot be reproduced anymore due to the linked commit adding JS that prevents empty emails from being submitted.
Of cause I can write a test reproducing this via the respective `POST` request and fix it, but the fix would simply be the fix for #629 imo. 
Not that I mind doing that, but since it's postponed I'd rather ask for clarification first :)

**Comment by original-brownbear** [comment]
@dmzaytsev ping :), I need some clarification of your wishes here given that this seems to be fixed as explained in the above.

**Comment by dmzaytsev** [comment]
@original-brownbear you're right, it [fixed already](https://github.com/yegor256/netbout/commit/442b8841ddf1aa0b25605960885461d1092be489)

@mkordas please close the ticket

**Comment by mkordas** [comment]
@dmzaytsev thanks

**Comment by original-brownbear** [comment]
@karato please pay attention to this ticket as well as my other closed tickets in your project. I closed all of them in a very short period of time each, still I'm blocked by having them pile up as open tasks and receive an unrealistic speed rating by them staying open long after the PRs are closed and payment on them is handled.
Thanks :)

**Comment by karato** [comment]
@elenavolokhova please, review this ticket for compliance with our [QA rules](http://at.teamed.io/qa.html)

**Comment by elenavolokhova** [comment]
@karato Quality is bad here as far as it was fixed in another issue.

**Comment by karato** [comment]
> @karato Quality is bad here as far as it was fixed in another issue.

@elenavolokhova thanks a lot, noticing bad quality is what makes us better

**Comment by karato** [comment]
@original-brownbear 10 mins was added to the account of @elenavolokhova (for QA review), in transaction 79383845
